src/main/java/discord4j/discordjson/json/OnboardingEditPromptOptionData.java

```diff
@@ -0,0 +1,41 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.Id;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.List;
+import java.util.Optional;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableOnboardingEditPromptOptionData.class)
+@JsonDeserialize(as = ImmutableOnboardingEditPromptOptionData.class)
+public interface OnboardingEditPromptOptionData {
```

Comment:
i dont fully check but this can apply for the same thing like OnboardingEditPromptData? not in the same form based in the emoji changes